portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,15 +33,32 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> comments;
-
-  @Override
-  public void init() {
-      comments = new ArrayList<>();
-  }
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    int userQuantity = getCommentQuantity(request);
+    
+    boolean all = true;
+    if (userQuantity != -1) all = false;
```

Comment:
you should always use { } with if statements, even if it's a single line. This was actually the cause of a famous apple security exploit a little while back: https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/

Reply:
Alternatively, i think you could use do boolean all = (userQuantity == -1);

Reply:
I've never actually used an if statement on a single line... for some reason I prefer the visual look of the code block below. This is definitely super helpful to note, thank you!